### DISCUSSION TITLE
Nearest Starbucks and consideration on passenger flow

### DISCUSSION THREAD

**Discussion by Stephenwzy** [jakobzhao/geog495#61] [Final Project]
Application: https://stephenwzy.github.io/final_project/index.html
Github: https://github.com/Stephenwzy/final_project

Description: The application shows the distribution of Starbucks coffee shops in Seattle City. The application will sort out the nearest Starbucks based on the geographic locations (nominal locations) provided by users in the search bar. The app offers two buttons on the index page (home page). The first one directs to "Visualization of raw visits," which displays the pattern of the sum of raw visits in each Starbucks from 2021/10/1 - 2021/11/1 based on POI data. The second one directs to "Visualization of dwelling time," which displays the pattern of visitors' median dwelling time in each Starbucks based on POI data.
![app](https://user-images.githubusercontent.com/46805422/145531258-1595e0de-2c8d-42d3-8144-1d28860c8ee4.png)

**Comment by notzulkifli** [comment]
Wow, this is actually outstanding, on top of making the index.html, you created two more html files upon clicking that will lead to the thematic map. I really like how you made each of the buttons clickable and each button serves a purpose. The only thing i can think of as a suggestion is to maybe space out the numbers on the bottom of your legend in dwelltime.html and visitorcounter.html so that the numbers in the end line up with the ends of the box. Amazing work!

**Comment by emilyphantastic** [comment]
This project is amazing! I'm so jealous of your skills. I love how well the theme is as well as how you basically made three maps that provided relevant information. I also love how the search bar allows users to search an area, and the closes Starbucks icon, pops up. A suggesting I have is to change the basemap style. Right now it's a very nice dark blue, but it gives me night time vibes. You know when you're driving at night on Apple Maps, the map turns dark after a certain time and turns bright after a certain time. I would personally like to see a brighter color to represent the morning since Starbucks locations are usually closed at night.

**Comment by yodapancake** [comment]
Great project! I think it's really fun to see where the Starbucks in Seattle are located especially since Starbucks was founded here but it's also fun to see which ones are most busy. It was also interesting seeing a webGIS applications with multiple webpages that required a navbar to navigate. In the future you could streamline your application by combining the three webpages into one somehow.

**Comment by whalen323** [comment]
Wow, this is awesome! I love the way you formatted the side panel text and colors and used a logo to match Starbucks! And it's also really cool that you even added some more filters that we can use to see different data like usage and distance! Honestly the only little thing I have to say is that maybe you could have formatted the phone numbers of the locations in a different way? Because to me it's not super obvious what this long string of numbers in the sidebar is. But overall, this is amazing!

**Comment by madiframpton** [comment]
Hi! This is a super useful map that compiles a lot of information into an easy-to-use application, nicely done!! I also really like the color scheme used here. It might be nice if the base-map contrasts with the points just a little bit more, but this is definitely not necessary.

**Comment by Zhiyuan-Jiao** [comment]
Hello. I really like the color scheme you chose for the base map and those little icon of Starbucks. They make really great contrast. One  thing I believe that could be improved on the web application is to add a sorting data by distance function so that the user could locate the closest Starbuck more quickly.
